config/minimal-config-example.toml

```diff
@@ -19,6 +19,13 @@ operator_mnemonic = "celery smart tip orange scare van steel radio dragon joy al
 # that is used by the `indexer-agent`. It is expected that `indexer-agent` will create
 # the necessary tables.
 postgres_url = "postgres://postgres@postgres:5432/postgres"
+# You can also use the following fields to specify the connection details separately.
```

Comment:
I understand you wanting to keep the same on both. Could you create a test if we have a `toml string` with the splitten config it's generating the expected PostgresVars variant?